### ISSUE TITLE
##exact-int->ratnum isn't inlined in _num.scm

### ISSUE THREAD

**Issue by gambiteer** [gambit/gambit#353]
`##exact-int->ratnum` isn't inlined in _num.scm.  I think that's because it's defined with `define-prim`.

It's used only in `_num.scm`.  Should we define it with simply `define` or make it a macro?

What about some other simple things? `##noncpxnum->cpxnum`? What else?

**Comment by gambiteer** [comment]
I guess these function already are just wrappers around macros.  How do you want to proceed?

**Comment by feeley** [comment]
```
(define-macro (macro-exact-int->ratnum x)
  (macro-ratnum-make ,x 1))
```

**Comment by feeley** [comment]
```
(define-macro (macro-noncpxnum->cpxnum x)
  `(macro-cpxnum-make ,x 0))
```

**Comment by gambiteer** [comment]
Fixed with 360b5aa7deb9a1951a7477096b1182818f01e41c
